Starter.Api/Startup.cs

```diff
@@ -1,41 +1,34 @@
-using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
+namespace Starter.Api;
 
-namespace Starter.Api
+public class Startup
 {
-    public class Startup
+    public IConfiguration Configuration { get; }
+
+    public Startup(IConfiguration configuration)
     {
-        public IConfiguration Configuration { get; }
+        Configuration = configuration;
+    }
 
-        public Startup(IConfiguration configuration)
-        {
-            Configuration = configuration;
-        }
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddControllers(o => { o.OutputFormatters.Clear(); })
+            .AddNewtonsoftJson(o => { o.UseCamelCasing(processDictionaryKeys: true); });
```

Comment:
Also, I don't remember why I used `NewtonsoftJson` here 😅. `System.Text.Json` should [support](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions?view=net-6.0#:~:text=DictionaryKeyPolicy,as%20camel%2Dcasing.) setting casing both for properties and dictionary keys. But this should be 100% tested before implementing

Reply:
There's no need to reference `NewtonsoftJson` or `System.Text.Json` anymore.
I tested the minimal api using the battlesnake CLI. See https://github.com/BattlesnakeOfficial/rules